person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? and @parent_permission
```

Comment:
- [ ] According to best practices as mentioned [here](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md), please don't use the `and` or `or` keywords. It's better to use `&&` and `||`
- [ ] According to the project requirements, this should be or(`||`) instead of and(`&&`.)